spec/features/admin/general_settings_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
```

Comment:
Also not sure about where to put this file. I treated it as a feature test, is that correct ?

Reply:
Yes, that's fine. If you wanted to test the whole feature, you would visit the root path `/` and test if the URL of the link is correct. That would be more apt for a feature test.